<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocTypeListView</class>
 <widget class="QWidget" name="DocTypeListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DocTypeListView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,0,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Support file </string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="extListWidget"/>
          </item>
          <item>
           <widget class="QListWidget" name="typListWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="subToolButton">
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addToolButton">
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Custom extension</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QListWidget" name="customListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>subToolButton</sender>
   <signal>clicked()</signal>
   <receiver>DocTypeListView</receiver>
   <slot>slot_sub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addToolButton</sender>
   <signal>clicked()</signal>
   <receiver>DocTypeListView</receiver>
   <slot>slot_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_sub()</slot>
  <slot>slot_add()</slot>
 </slots>
</ui>
